app/build.gradle

```diff
@@ -541,6 +541,7 @@ dependencies {
     implementation Deps.androidx_datastore
     implementation Deps.protobuf_javalite
     implementation Deps.google_material
+    implementation Deps.google_guava
```

Comment:
We normally try to not include external dependencies unless it's strictly necessary, for multiple reason and as they contribute to increase the application size.

Reply:
Find in [my other comment](https://github.com/mozilla-mobile/fenix/pull/22455#discussion_r756729134) my reasoning about why I ended up including Guava.
I agree with the concerns, another one is that this part of the Guava API is beta. This was one of my reasons to add this behind a feature flag.
Let's make a decision in the other thread.

(If size is the main blocker at the end, I can compare the size of the apks with and without this dependency).